[2025-10-03T00:31:03.555+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-03T00:31:03.597+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_taskk.task_prnt scheduled__2025-10-03T00:30:00+00:00 [queued]>
[2025-10-03T00:31:03.612+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_taskk.task_prnt scheduled__2025-10-03T00:30:00+00:00 [queued]>
[2025-10-03T00:31:03.613+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-10-03T00:31:03.645+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): task_prnt> on 2025-10-03 00:30:00+00:00
[2025-10-03T00:31:03.661+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1031) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-03T00:31:03.664+0000] {standard_task_runner.py:63} INFO - Started process 1033 to run task
[2025-10-03T00:31:03.666+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'first_taskk', 'task_prnt', 'scheduled__2025-10-03T00:30:00+00:00', '--job-id', '285', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmpdz5gzjz7']
[2025-10-03T00:31:03.673+0000] {standard_task_runner.py:91} INFO - Job 285: Subtask task_prnt
[2025-10-03T00:31:03.698+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-10-03T00:31:03.784+0000] {task_command.py:426} INFO - Running <TaskInstance: first_taskk.task_prnt scheduled__2025-10-03T00:30:00+00:00 [running]> on host 452c0657f430
[2025-10-03T00:31:03.966+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='first_taskk' AIRFLOW_CTX_TASK_ID='task_prnt' AIRFLOW_CTX_EXECUTION_DATE='2025-10-03T00:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-03T00:30:00+00:00'
[2025-10-03T00:31:03.974+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-03T00:31:03.976+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-10-03T00:31:03.979+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'date']
[2025-10-03T00:31:04.006+0000] {subprocess.py:86} INFO - Output:
[2025-10-03T00:31:04.016+0000] {subprocess.py:93} INFO - Fri Oct  3 00:31:04 UTC 2025
[2025-10-03T00:31:04.018+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-10-03T00:31:04.020+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-03T00:31:04.086+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=first_taskk, task_id=task_prnt, run_id=scheduled__2025-10-03T00:30:00+00:00, execution_date=20251003T003000, start_date=20251003T003103, end_date=20251003T003104
[2025-10-03T00:31:04.163+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-03T00:31:04.214+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-10-03T00:31:04.217+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
